Python/Python.sublime-syntax

```diff
@@ -232,27 +403,30 @@ contexts:
     - match: '='
       scope: keyword.operator.assignment.python
       push:
+        - meta_scope: meta.function.default-value.python
```

Comment:
Imo meta scopes shouldn't be restricted in precision for a reason like this. We are still in the parameters "context" after all and just entered an even more specific meta scope, so the older one should remain.

Same for the annotation comment below.